spotify/main.py

```diff
@@ -210,11 +221,14 @@ def main():
         continueOnError=True,
     )
 
+    # Calculate the number of days between start and end date
+    days = (end_datetime - start_datetime).days
```

Comment:
would move this to the beginning of the file to have all date calculations, setting default values, etc. in one spot